db/migrate/20220402165156_devise_create_users.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email, null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
+
+      ## Recoverable
+      t.string :reset_password_token
+      t.datetime :reset_password_sent_at
```

Comment:
I think you disabled recoverable module. So, you do not need these columns in the database.

```suggestion
```

Reply:
Yes of course, didn't think about it